localstack/services/lambda_/event_source_listeners/lambda_legacy.py

```diff
@@ -0,0 +1,11 @@
+# TODO: remove this legacy construct when re-working event source mapping.
```

Comment:
This should be re-worked post v3 but I tried to keep this removal PR minimal and avoid functional changes